scripts/update_license_headers.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+import re
+import sys
+
+def update_license_header(file_path):
+    with open(file_path, 'r') as file:
+        content = file.read()
+
+    apache_header_pattern = re.compile(
+        r'(/\*|#|//)\s*Licensed under the Apache License, Version 2\.0.*?'
```

Comment:
why `|#|` ?

Reply:
Thanks for spotting the |#|. We're only dealing with Go files, so we only need // for comments. I'll simplify the regex to just look for that.